src/main/java/com/abi/quotes/views/chat_detail/ChatDetailView.java

```diff
@@ -249,14 +268,22 @@ private Component createMessageBox(JsonObject message) {
 		timeSpan.getStyle().set("font-size", "10px");
 
 		bubbleLayout.add(senderSpan, messageText);
-		
-		for (int i = 0; i < attachments.size(); i++) {
-			bubbleLayout.add(new AttachmentDisplay(attachments.get(i).getAsJsonObject()));
+
+		if (attachments.size() > 0) {
+			// Copied from BetaTestView
+			UI.getCurrent().getPage().retrieveExtendedClientDetails(receiver -> {
+				screenWidth = receiver.getScreenWidth();
+				//System.out.println("Screen width: " + screenWidth);
+
+				for (int i = 0; i < attachments.size(); i++) {
+					bubbleLayout.add(new AttachmentDisplay(attachments.get(i).getAsJsonObject(), screenWidth));
+				}
+
+				bubbleLayout.add(timeSpan);
+				bubbleLayout.setHorizontalComponentAlignment(Alignment.END, timeSpan);
+			});
```

Comment:
Doing so, hides the timeSpan if there are no attachments. Don't change it, I'll do so after merging.